codice/alliance
# CAL-10 - Create thumbnails when ingesting a NITF

#### What does this PR do?
This PR adds thumbnails to ingested NITF products.

#### Who is reviewing it?
@kcwire 
@beyelerb 
@jlcsmith 

#### How should this be tested?
1. Install and start the alliance imaging application.
2. Log into DDF
3. Ingest a NITF
4. Do an empty search
    - verify that a thumbnail appears for the ingested NITF.

#### Any background context you want to provide?
#### What are the relevant tickets?
CAL-10

#### Screenshots (if appropriate)
![thumbnail-demo](https://cloud.githubusercontent.com/assets/5922390/14034894/027cc55c-f1e5-11e5-8707-1036e7111759.png)

#### Checklist:

Cal 10 - Create Nitf Input Transformer
